<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LearnWordConfigDialog</class>
 <widget class="QDialog" name="LearnWordConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>    配置   </string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/new/prefix1/rc/actions/gear.png</normaloff>:/new/prefix1/rc/actions/gear.png</iconset>
  </property>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/prefix1/rc/actions/apply.png</normaloff>:/new/prefix1/rc/actions/apply.png</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="ConfigGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>选项</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="ConfigTabWidget">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <widget class="QWidget" name="LearnTab">
       <attribute name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/new/prefix1/rc/actions/configure_shortcuts.png</normaloff>:/new/prefix1/rc/actions/configure_shortcuts.png</iconset>
       </attribute>
       <attribute name="title">
        <string>学习模式</string>
       </attribute>
       <widget class="QGroupBox" name="LearnMethodGroupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>111</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>学习方式</string>
        </property>
        <widget class="QComboBox" name="LearnMethodComboBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <item>
          <property name="text">
           <string>出词忆义</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>出义忆词</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>听音辨词</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="LearnLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>学习方式选择：</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="OutGroupBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>出词顺序</string>
        </property>
        <widget class="QRadioButton" name="OutOrderlyRadioButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>顺序</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="OutRandomRadioButton">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>20</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>随机</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="ReviewGroupBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>70</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>复习顺序</string>
        </property>
        <widget class="QRadioButton" name="ReviewOrderlyRadioButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>顺序</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="ReviewRandomRadioButton">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>20</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>随机</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="SoundTab">
       <attribute name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/new/prefix1/rc/actions/speaker.png</normaloff>:/new/prefix1/rc/actions/speaker.png</iconset>
       </attribute>
       <attribute name="title">
        <string>语音</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QGroupBox" name="SpeakMethodeGroupBox">
          <property name="title">
           <string>发音模式设定</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QRadioButton" name="TTSRadioButton">
             <property name="text">
              <string>内置或第三方TTS</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QRadioButton" name="OnlineRadioButton">
             <property name="text">
              <string>Google在线TTS</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QRadioButton" name="Real_TTSRadioButton">
             <property name="text">
              <string>真人语音配合TTS</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="CircleGroupBox">
          <property name="title">
           <string>语音循环设定</string>
          </property>
          <widget class="QLabel" name="TimeLabel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>60</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>循环次数：</string>
           </property>
          </widget>
          <widget class="QComboBox" name="TimeComboBox">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>30</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
          </widget>
          <widget class="QLabel" name="IntervalLabel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>60</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>循环间隔：</string>
           </property>
          </widget>
          <widget class="QComboBox" name="IntervalComboBox">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>70</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>500</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1300</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2000</string>
            </property>
           </item>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ShortcutsTab">
       <attribute name="icon">
        <iconset resource="rc.qrc">
         <normaloff>:/new/prefix1/rc/actions/transform_move.png</normaloff>:/new/prefix1/rc/actions/transform_move.png</iconset>
       </attribute>
       <attribute name="title">
        <string>快捷键</string>
       </attribute>
       <widget class="QLabel" name="RememLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>记得：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="RememLineEdit">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="ObscureLabel">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>模糊：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ObscureLineEdit">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="ForgetLabel">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>20</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>忘记：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ForgetLineEdit">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>20</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="RightLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>61</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>正确：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="RightLineEdit">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>60</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="WrongLabel">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>61</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>错误：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="WrongLineEdit">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>60</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="SoundLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>101</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>语音：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="SoundLineEdit">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>100</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="NextLabel">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>100</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>下个：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="NextLineEdit">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>100</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="ConfigLabel">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>100</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>配置：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ConfigLineEdit">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>100</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="EditWordLabel">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>61</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>编辑：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="EditWordLineEdit">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>60</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="QuitButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/prefix1/rc/actions/dialog_cancel.png</normaloff>:/new/prefix1/rc/actions/dialog_cancel.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
